### PR TITLE
Cart page and ifxed navbar css

### PR DESCRIPTION
**Pull request by tarunsankhla**

### CODE REVIEW

File: src/CartPage/CartPage.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.cart-aside-footer{
+    padding: 1rem;
+}
+
+.cart-aside-footer > button{
+    width: 100%;
+    border: none;
+    background-color: rgb(117, 117, 204);
```
Comment (rahulyadav139) [inline src/CartPage/CartPage.css:47]: Declare colour in a central CSS file.

File: src/CartPage/CartPage.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+.cart-aside-footer > button:hover{
+    transform: scale(0.98);
+    background-color: rgb(53, 53, 80);
+}
+
+.cart-banner-stretch{
+    width: 100%;
+    height: fit-content;
+    flex-wrap: wrap;
+    /* height: 18rem; */
```
Comment (rahulyadav139) [inline src/CartPage/CartPage.css]: Remove unnecessary comments.

**Review by rahulyadav139 — APPROVED** [review]

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 🚀

File: style.css [hunk 3]
```diff
@@ -20,21 +21,45 @@ html{
...
+body::-webkit-scrollbar-thumb{
+    background-color: var(--btn-text-color-2);
+    border-radius: 1rem;
+    border: 3px solid  var(--shade-1);
+}
+
+body::-webkit-scrollbar:hover{
+    transform: scale(1.2);
+    cursor: pointer;
+}
```
Comment (m26unkwn) [inline style.css:39]: This is great giving styling to the scrollbar.

**Review by m26unkwn — APPROVED** [review]
For me, it's good to merge.

File: src/CartPage/CartPage.css (hunk 2 above)
Comment (tarunsankhla) [reply src/CartPage/CartPage.css]: done

**Pull request merged**